packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -54,6 +55,35 @@ export default class OperatorModeStateService extends Service {
     this.schedulePersist();
   }
 
+  patchCard = task({ enqueue: true }, async (id: string, attributes: any) => {
+    let stackItems = this.state?.stacks.flat() ?? [];
+    for (let item of stackItems) {
+      if ('card' in item && item.card.id == id) {
+        await this.setFieldValues.perform(item.card, attributes);
+      }
+    }
+  });
+
+  private setFieldValues = task(async (card: CardDef, values: any) => {
+    let fields = await this.cardService.getFields(card);
+    for (let fieldName of Object.keys(values)) {
+      let field = fields[fieldName];
+      if (fieldName === 'id') continue;
+      if (
+        (field.fieldType === 'contains' ||
+          field.fieldType === 'containsMany') &&
+        !(await this.cardService.isPrimitive(field.card))
+      ) {
+        await this.setFieldValues.perform(
+          (card as any)[fieldName],
+          values[fieldName],
+        );
+      } else {
+        (card as any)[fieldName] = values[fieldName];
+      }
+    }
+  });
```

Comment:
we might want to consider refactoring this into the card-api.ts

Reply:
is this meant to operator on linked fields too? if not you should throw an error if you hit that case

Reply:
Yeah this caused the issue with the dates, I had a pr around deserialising all of the fields. The flow of serialize -> merge -> deserialise was one I'd considered but wasn't sure of the right parts of the API to use, would be good to pair on that. I also wanted that instead as it'd be good to validate the entire change as one.

Reply:
I think ultimately you’ll just end up reinventing our card deserializer, so let’s just use that.